unit_test/transforms/test_mapper.py

```diff
@@ -94,3 +94,20 @@ def test_replace_joint_in_connected_assets_error(self):
         # assert
         self.assertIsInstance(cm.exception, RuntimeError)
         self.assertEqual(cm.exception.args[0], "Error in replacing joint in connected assets.")
+
+    def test_component_with_4_connection_points(self):
+        """Method to test the to entity mapper class with 4 connection points in a asset."""
+        esdl_file_path = Path(__file__).parent / ".." / ".." / "testdata" / "simple_heat_pump.esdl"
+        esdl_file_path = str(esdl_file_path)
+        esdl_object = EsdlObject(pyesdl_from_file(esdl_file_path))
+        network = Network()
+
+        # arrange
+        result = EsdlEnergySystemMapper(esdl_object).to_entity(network)
```

Comment:
result should be py_asset_list, py_junction_list.

Reply:
Je zou ook assertListEqual kunnen gebruiken om te checken of alle assets erin zitten? De huidige assert statements zijn vaag, laten we even kijken of dit iets duidelijker kan.

Reply:
Check connectivity of all components.

Reply:
eerste twee aangepast derde moet nog even kijken heo dat handig te doen

Reply:
For the third point I created separate issue #165 to create this functionality